scripts/misc/s3_generate_input.py

```diff
@@ -0,0 +1,41 @@
+"""
```

Comment:
We probably don't need this script since this is just `STEPS=generate_input python main.py`.